android/src/main/java/com/urbanairship/reactnative/UrbanAirshipReactModule.java

```diff
@@ -180,6 +180,15 @@ public void setUserNotificationsEnabled(boolean enabled) {
         UAirship.shared().getPushManager().setUserNotificationsEnabled(enabled);
     }
 
+    /**
+     * Enables channel creation if `channelCreationDelayEnabled` was
+     * enabled in the config.
```

Comment:
This is the method you want apps to call when they are ready to enable channel creation, right? So maybe the comment might be something like:
```
    * If `channelCreationDelayEnabled` is enabled in the config, apps must call 
    * this method to enable channel creation.
```